### DISCUSSION TITLE
Why some endpoint can not be  searched in trace page

### DISCUSSION THREAD

**Discussion by czxin788** [apache/skywalking#8953] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/13861904/165243648-98cae83a-538b-4bfe-8697-300d02d709e9.png)

Why some endpoint can not be  searched in  input dialog box of trace page?

**Comment by wu-sheng** [comment, marked as answer]
What do you mean can't?

**Reply by wu-sheng** [reply]
Where is your normal from?

**Reply by czxin788** [reply]
In the service Edu-Test::universal, it has a endpoint named  "Balancer/cms/column/config/course/v4/getColumnCourseListForCard",
![image](https://user-images.githubusercontent.com/13861904/165247136-38d4f42c-00ee-499f-85d9-9e0f61e09a52.png)

But I put "Balancer/cms/column/config/course/v4/getColumnCourseListForCard" in the endpoint dialog box,  there are nothing  which named  "Balancer/cms/column/config/course/v4/getColumnCourseListForCard" of endpoint.
![image](https://user-images.githubusercontent.com/13861904/165247701-2f5a0342-3635-4442-9d72-6d230123ad03.png)

**Reply by wu-sheng** [reply]
If you are using ElasticSearch, try to learn what is ElasticSearch word analyzer, what is stop word.

**Reply by czxin788** [reply]
Thanks
